resources/lang/en/beatmaps.php

```diff
@@ -124,6 +126,10 @@
             'wip' => 'Note: This beatmap is marked as a work-in-progress by the creator.',
         ],
 
+        'votes' => [
+            'up' => 'Upvote this discussion',
```

Comment:
I don't think this helps because the problem is it isn't clear that voting up/down affects kudosu.